bin/libs/zos-jes.ts

```diff
@@ -98,95 +98,96 @@ export function submitJob(jclFileOrContent: string, printJobDebug:boolean=true,
   }
 }
 
-export function waitForJob(jobid: string): {jobcctext?: string, jobcccode?: string, jobid?: string, jobname?: string, rc: number} {
-  let jobstatus;
-  let jobname;
-  let jobcctext;
-  let jobcccode;    
-  let is_jes3;
+/**
+ * Returns job name, completion code, and status. If we cannot retrieve the completion code, it is set to -1. 
```

Comment:
`If we cannot retrieve the completion code, it is set to -1. `
The default compCode is empty string (L113).

Reply:
Updated to match current behavior.